src/hooks/usePlayList.js

```diff
@@ -4,38 +4,47 @@ import getPlaylistData from '../utils/getPlaylistData'
 import getWrongAnswers from '../utils/getWrongAnswers'
 import shuffle from '../utils/shuffleArray'
 
-export default function usePlayList(playlist) {
+export default function usePlayList(initialPlaylist) {
+  const [playlist, setPlaylist] = useState(initialPlaylist)
   const [playlistData, setPlaylistData] = useState(null)
   const [answers, setAnswers] = useState(null)
   const [counter, setCounter] = useState(null)
   const [wrongAnswers, setWrongAnswers] = useState(null)
+  const [isLoaded, setIsLoaded] = useState(false)
```

Comment:
I maybe works just like this:
```suggestion
  const isLoaded = playlist && counter && answers
```